tests/fixture_valid.js

```diff
@@ -80,13 +80,55 @@ module.exports.checkFixture = function checkFixture(fixture, usedShortNames=[])
       }
 
       for (const ch of mode.channels) {
-        if (!(ch in fixture.availableChannels) && ch !== null) {
-          result.errors.push({
-            description: `channel '${ch}' referenced from mode '${modeShortName}' (#${i}) but missing.`,
-            error: null
-          });
-        }
         usedChannels.push(ch);
+
+        if (ch !== null) {
+          if (!(ch in fixture.availableChannels) && ch !== null) {
+            result.errors.push({
+              description: `channel '${ch}' referenced from mode '${modeShortName}' (#${i}) but missing.`,
+              error: null
+            });
+          }
+          else {
+            switch (fixture.availableChannels[ch].type) {
+              case 'Pan':
+                checkPanTiltMaxExistence(mode, ch, 'panMax');
+                break;
+
+              case 'Tilt':
+                checkPanTiltMaxExistence(mode, ch, 'tiltMax');
+                break;
+            }
+          }
+        }
+      }
+    }
+
+    function checkPanTiltMaxExistence(mode, chKey, panTiltMax) {
+      let maxDefined = false;
+      let maxIsNull = false;
+      if ('physical' in mode
+        && 'focus' in mode.physical
+        && panTiltMax in mode.physical.focus) {
+        maxDefined = true;
+        maxIsNull = mode.physical.focus[panTiltMax] === 0;
+      }
+      else if ('physical' in fixture
+        && 'focus' in fixture.physical
+        && panTiltMax in fixture.physical.focus) {
+        maxDefined = true;
+        maxIsNull = fixture.physical.focus[panTiltMax] === 0;
+      }
+
+      const chType = fixture.availableChannels[chKey].type;
+      if (!maxDefined) {
+        result.warnings.push(`${panTiltMax} is not defined although there's the ${chType} channel '${chKey}'`);
+      }
+      else if (maxIsNull) {
+        result.errors.push({
+          description: `${panTiltMax} is 0 in mode '${mode.name || mode.shortName}' although it has the ${chType} channel '${chKey}'`,
```

Comment:
For other warnings/errors, we used the `const modeShortName = mode.shortName || mode.name;` variable, so I decided to use the same name.